src/main/java/frc/robot/Constants.java

```diff
@@ -21,6 +24,10 @@ public final class Constants {
 
       public static final double FIELD_WIDTH_METERS = 16;
       public static final double FIELD_HEIGHT_METERS = 7.85;
```

Comment:
I added a unified Pose2d object which we could reference in case the simulator is wrong. I would like to see how the code runs as it is.